twisted/protocols/tls.py

```diff
@@ -379,23 +385,12 @@ def _flushReceiveBIO(self):
                 self._tlsShutdownFinished(None)
             except Error as e:
                 # Something went pretty wrong.  For example, this might be a
-                # handshake failure (because there were no shared ciphers, because
-                # a certificate failed to verify, etc).  TLS can no longer proceed.
-
-                # Squash EOF in violation of protocol into ConnectionLost; we
-                # create Failure before calling _flushSendBio so that no new
-                # exception will get thrown in the interim.
-                if e.args[0] == -1 and e.args[1] == 'Unexpected EOF':
-                    failure = Failure(CONNECTION_LOST)
-                else:
-                    failure = Failure()
```

Comment:
It's not gone, just refactored (there's test coverage for this error translation, and if you remove it they fail).  See https://github.com/twisted/twisted/pull/191/files#diff-1f9547fad75f119c2521dbaa75d86ae1R470